fidesctl/.fides/dataset.yml

```diff
@@ -55,6 +62,10 @@ dataset:
           - name: parent_key
             description: "Fides Generated Description for Column: parent_key"
 
+          - name: *created_at
```

Comment:
Added some anchor/alias business here when updating our current `dataset.yml` - it will get wiped out by `annotate-dataset` however so probably need to remove/rethink using these 🤔

Reply:
Planning to add a follow on issue for these to not hold up working through the remaining model items

Reply:
yeah might as well remove them now 🤷

Reply:
although we update our stuff by hand, we also are an example for the community and I don't want them to think that this works if it doesn't really

Reply:
Will do 👍🏽  my bad, forgot these were still in here